R/adjust-numeric-calibration.R

```diff
@@ -1,8 +1,11 @@
 #' Re-calibrate numeric predictions
 #'
 #' @param x A [container()].
-#' @param calibrator A pre-trained calibration method from the \pkg{probably}
-#' package, such as [probably::cal_estimate_linear()].
+#' @param type Character. One of `"linear"`, `"isotonic"`, or
```

Comment:
When putting together #13, ultimately felt that this should have a different argument than the same one supplied to `container(type)`. I'd propose `method`, but will wait to Replace All until there's more feedback here.